packages/indexer-cli/src/commands/indexer/actions/get.ts

```diff
@@ -1,7 +1,8 @@
+import { ActionOrderByInput } from './../../../../../indexer-common/src/actions'
```

Comment:
Move this import to be from @graphprotocol/indexer-common? (currently on line 5)